src/main/webapp/js/calendar.js

```diff
@@ -13,22 +13,187 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-function initCalendar() {
-  new tui.Calendar('#calendar', {
-    defaultView: 'week',
-    useCreationPopup: true,
-    useDetailPopup: true,
-    disableDblClick: true,
-    disableClick: true,
-    isReadOnly: true,
-    scheduleView: ['time'],
-    taskView: false,
-    week: {
-      workweek: true,
-    },
+// TODO(savsa): Add ability to remove courses from the calendar
+// TODO(#34): Add ability to choose which section the user wants, not just
+//  automatically choosing the first section
+
+export const Calendar = (() => {
+  let id = 1;  // The id of the next scheudle that will be added to the calendar
+  let cal;     // The calendar object
+  const enum_days = {
+    DATE_MONDAY: 0,
+    DATE_TUESDAY: 1,
+    DATE_WEDNESDAY: 2,
+    DATE_THURSDAY: 3,
+    DATE_FRIDAY: 4,
```

Comment:
I haven't seen anything on the weekend yet and courseoff.com doesn't show the weekend either. Maybe @naaoli can shed some light on this.

Reply:
It think it's pretty simple to add the enum values for Saturday/Sunday just in case. I don't think there's any harm in doing that. Alternatively, we can choose throw an error on the backend if we see a course that happens on the weekend (before handing it off to the frontend). I just want to make sure we handle these edge cases, since that's an important part of good software design.